CHANGES.md

```diff
@@ -0,0 +1,6303 @@
+- [#2919](https://github.com/gap-system/gap/pull/2919) The undocumented (!) function `SetFeatureObj` is obsolete; use `SetFilterObj` resp. `ResetFilterObj` instead.
+- [#3185](https://github.com/gap-system/gap/pull/3185) `(Un)HideGlobalVariables` is obsolete
+- [#3269](https://github.com/gap-system/gap/pull/3269) `TemporaryGlobalVarName` is obsolete
+- [#3409](https://github.com/gap-system/gap/pull/3409) Remove `BANNER`, `QUIET`
+
+## Packages
+
+- [#3215](https://github.com/gap-system/gap/pull/3215) Issue tracker, maintainers and contributors are printed in package banners
+- [#3286](https://github.com/gap-system/gap/pull/3286) Teach `ValidatePackageInfo` about the optional `License` field
+
+## Other changes
+
+- [#2709](https://github.com/gap-system/gap/pull/2709) Rename `MultRowVector` to `MultVector` (the old name is still supported, but marked as obsolete)
+- [#2729](https://github.com/gap-system/gap/pull/2729) Rename `QuaternionGroup` to `DicyclicGroup`, document `IsDihedralGroup` and `IsQuaternionGroup`
+- [#3010](https://github.com/gap-system/gap/pull/3010) Read `lib/transatl.g` before any `gap.ini` file, when GAP is loaded
+- [#3406](https://github.com/gap-system/gap/pull/3406) Remove RXVT-shell support for Windows
+- [#3480](https://github.com/gap-system/gap/pull/3480) `BuildPackages.sh` now executes `make clean` before full build
+
+## New packages redistributed with GAP
+- [CddInterface](https://homalg-project.github.io/CddInterface/) GAP interface to cdd, by Kamal Saleh
+- [DifSets](https://dylanpeifer.github.io/difsets/)  Enumeration of the difference sets (up to equivalence) in 
+ groups,  by Dylan Peifer
+- [ferret](https://gap-packages.github.io/ferret/) C++ reimplementation of Jeffery Leon’s Partition Backtrack framework for solving problems in permutation groups, by Christopher Jefferson
+- [images](https://gap-packages.github.io/images/) Finding minimal and canonical images in permutation groups, by Christopher Jefferson, Markus Pfeiffer, Rebecca Waldecker, Eliza Jonauskyte
+- [NCOnvex](https://homalg-project.github.io/NConvex/) Polyhedral constructions and computations for cones, polyhedrons, polytopes and fans, by Kamal Saleh, Sebastian Gutsche
+- [NoCK](https://pjastr.github.io/NoCK/) Computation of Tolzanos’s obstruction for compact Clifford-Klein forms, by Maciej Bocheński, Piotr Jastrzębski, Anna Szczepkowska, Aleksy Tralle, Artur Woike
+
+### 2 Changes between GAP 4.9 and GAP 4.10
+
+This chapter contains an overview of the most important changes
+introduced in GAP 4.10.0 release (the first public release of
+GAP 4.10), as well as about subsequent update releases for GAP 4.10.
+
+These changes are also listed on the Wiki page
+<https://github.com/gap-system/GAP/wiki/gap-4.10-release-notes>.
```

Comment:
I would remove everything from "### 2 Changes between GAP 4.9 and GAP 4.10" till here, I don't see any useful information.